go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/ChainSafe/gossamer
 
 require (
-	github.com/ChainSafe/chaindb v0.1.5-0.20220322154826-c0d431995732
+	github.com/ChainSafe/chaindb v0.1.5-0.20220708005902-df45dbc8e840
```

Comment:
Did we need to bump this up?

Reply:
Yea I wanted to remove the trace logs from badger.

Reply:
Thank god 👍 although we should *ideally* have this as a separate PR

Reply:
This PR is merging into [tim/v0.9.20-fix-ci](https://github.com/ChainSafe/gossamer/tree/tim/v0.9.20-fix-ci). I'll make a PR for this change.